src/test/java/com/google/devtools/build/lib/skyframe/FileArtifactValueTest.java

```diff
@@ -256,7 +256,7 @@ public void testUptodateCheck() throws Exception {
     clock.advanceMillis(1);
     assertThat(value.wasModifiedSinceDigest(path)).isFalse();
     clock.advanceMillis(1);
-    path.setLastModifiedTime(123); // Changing mtime implicitly updates ctime.
```

Comment:
I think we should keep this test case. I will create another test case for modified content when importing.